src/hotel-management-app/src/app/customer/payment/payment.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { CustomerService } from '../../services/customer.service';
+import { CartService } from '../../services/cart.service';
+
+@Component({
+  selector: 'app-payment',
+  templateUrl: './payment.component.html',
+  styleUrls: ['./payment.component.css']
+})
+export class PaymentComponent {
+  paymentDetails = {
```

Comment:
Can you create typecasting for this variable and if it's a globally used type have in a separate file and import where you need to use it

Reply:
resolved